daml-lf/spec/daml-lf-1.rst

```diff
@@ -2005,17 +2005,17 @@ types that satisfies the following rules::
     n <ₜ ⟨ f₁ : τ₁, …, fₘ : τₘ ⟩
 
     fᵢ comes lexicographically before gᵢ
-  ——————————————————————————————————————————————————— TypeOrderStructFieldName
-    ⟨ f₁ : τ₁, …, fₘ : τₘ ⟩ <ₜ
-      ⟨ f₁ : σ₁, …, fᵢ₋1 : σᵢ₋₁, gᵢ : σᵢ, …, gₙ : σₙ ⟩
+  ——————————————————————————————————————————————————— TypeOrderStructFieldNameDiff
+    ⟨  f₁ : τ₁, …, fᵢ₋₁ : τᵢ₋₁, fᵢ : τᵢ, …, fₘ : τₘ ⟩ <ₜ
+        ⟨  f₁ : σ₁, …, fᵢ₋₁ : σᵢ₋₁, gᵢ : σᵢ, …, gₘ : σₘ⟩
```

Comment:
Wouldn't it be easier to say something like
```
  [f_1, ..., f_m] is lexicographically less than [g_1, ..., g_n]
------------------------------------------------------------------
  <f_1: s_1, ..., f_m: s_m> <_t <g_1: t_1, ..., g_n: t_n>
```
?

Reply:
It is simpler. We agree with @cocreature to go this way. 
Note the resulting order is not the same. It is somehow less natural as it does not behave like map.